### DISCUSSION TITLE
Link to the old Google Discussion Group

### DISCUSSION THREAD

**Discussion by rmcdermo** [firemodels/fds#12653] [Announcements]
[FDS Google Discussion Group](https://groups.google.com/g/fds-smv) (frozen on March 20, 2024)

**Comment by Selvahasan** [comment]
Hi

**Comment by Selvahasan** [comment]
Please link in this group

**Reply by TristanHehnen** [reply]
Could you please clarify what you mean?
 
In Randy's original post above there is a link to the old Google Group. 
In said group, there is a post by Kevin about retiring the group and linking here ([Good-bye GoogleGroups](https://groups.google.com/g/fds-smv/c/ChmQ1WTs9eQ)).
So both groups link to each other already.

**Reply by Selvahasan** [reply]
Hi

**Comment by shanmugasundaram2706** [comment]
Hai Developers,
               Could you please clarify me how the char oxidation rate is calculated if the "SURFACE OXIDATION MODEL" is kept FALSE? How the oxygen mass fraction at the fuel surface is calculated?  Does the particle be considered as thermally thick? 

Thanks in advance,

Regards,
Shanmugasundaram D

**Reply by mcgratta** [reply]
Start a new thread. This is an old announcement.
